tensorflow/compiler/xla/client/lib/math.cc

```diff
@@ -341,7 +341,10 @@ XlaOp ErfInv(XlaOp x) {
       0.00943887047f,   1.00167406f,     2.83297682f};
 
   auto one = ScalarLike(x, 1.0);
-  auto w = -Log((one - x) * (one + x));
+  // Compute logarithm of (1+arg).  (This function is more precise than
```

Comment:
Nit: Compute logarithm of (1+arg) using log1p(arg) which is more precise than log(1+arg) when arg is close to zero.
See ref. https://en.cppreference.com/w/cpp/numeric/math/log1p

Reply:
Thank you @bixia1.